lms/envs/common.py

```diff
@@ -657,7 +657,17 @@
     # .. toggle_tickets: https://github.com/edx/edx-platform/pull/7315
     'LICENSING': False,
 
-    # Certificates Web/HTML Views
+    # .. toggle_name: FEATURES['CERTIFICATES_HTML_VIEW']
```

Comment:
it looks like the other features only include the name, no need to mention they are keys

Reply:
@eLRuLL All the feature flag names are set as keys for `FEATURES`. e.g: https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L81

Reply:
@msaqib52 oops, I don't know what I was looking at but you are totally right, sorry about that